### PR TITLE
feat: mech metrics

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Added metrics for mech page: predict, contribute, governatooor and other.

## Screenshots/Recordings

Before:
image
After:
image

### CODE REVIEW

File: components/MechPage/MechAgentMetrics.jsx [hunk 1]
```diff
@@ -1,24 +1,159 @@
...
+  return {
+    totalRqs: result.totalRqs,
+    predictRqs: result.predictRqs,
+    contributeRqs: result.contributeRqs,
+    governatooorRqs: result.governatooorRqs,
+    otherRqs: result.otherRqs,
+  };
+};
+
+const usePersistentSWR = (key, fetcher) =>
```
Comment (truemiller) [inline components/MechPage/MechAgentMetrics.jsx:21]: can be made reusable, but minor

**Review by truemiller — APPROVED** [review]

File: components/MechPage/MechAgentMetrics.jsx (hunk 1 above)
Comment (Tanya-atatakai) [reply components/MechPage/MechAgentMetrics.jsx:21]: i'd rather switch to react tanstack like in mono repo

File: common-util/constants.js [hunk 2]
```diff
@@ -21,10 +21,13 @@ export const CORE_TECHNICAL_DOCUMENT =
 export const FLIPSIDE_URL =
   'https://flipsidecrypto.xyz/flipsideteam/olas-key-activity-metrics-pnPjda';
 export const DUNE_QUERY_URL = 'https://dune.com/queries';
-export const PREDICTION_DAA_QUERY_ID = '4165113';
+export const PREDICTION_DAA_QUERY_ID = '4185253';
```
Comment (Tanya-atatakai) [inline common-util/constants.js:24]: was this changed accidentally? the value looks the same as mech query's, but I think should be the old one, the metric is also broken in Vercel

**Review by Tanya-atatakai — CHANGES_REQUESTED** [review]

File: common-util/constants.js (hunk 2 above)
Comment (atepem) [reply common-util/constants.js:24]: Yes, I think it was changed on accident. I'll change this in a new PR real quick

**Pull request merged**